.changeset/small-snakes-guess.md

```diff
@@ -0,0 +1,15 @@
+---
+"@latticexyz/common": patch
```

Comment:
```suggestion
```
I don't think this changeset makes much sense looking at this package alone

If we want a changeset for common, should add one that is just something like "Added export for root namespace string" with an example of how to use it

Reply:
100% agree